src/App.scss

```diff
@@ -0,0 +1,341 @@
+    width: 100%;
+    background-color: $white;
+    border: solid 1px $gray;
+
+    .book {
+      .category {
+        opacity: 0.5;
+        font-size: 0.875rem;
+        font-weight: bold;
+        color: $black;
+      }
+
+      .title,
+      .author,
+      .actions button {
+        font-family: 'Roboto Slab', serif;
+      }
+
+      .title {
+        font-size: 1.375rem;
+        font-weight: bold;
+        letter-spacing: -0.2px;
+      }
+
+      .author {
+        font-size: 0.875rem;
+        font-weight: 300;
+        color: $bluish;
+      }
+
+      .actions {
+        display: flex;
+        gap: 0.5rem;
+        list-style: none;
+        margin-top: 1rem;
+
+        .vertical-line {
+          width: 0.125rem;
+          height: 1.5rem;
+          background-color: $gray;
+          margin: 0 1rem;
+        }
+
+        button {
+          background: none;
+          border: none;
+          color: $bluish;
+          font-size: 0.875rem;
+          font-weight: 300;
+          transition: color 0.1s ease-in, font-weight 0.1s ease-in;
+          cursor: pointer;
+
+          &:hover {
+            font-weight: 500;
+            color: $blue;
+          }
+        }
+      }
+    }
+
+    .progress-container {
+      display: flex;
+      align-items: center;
+      gap: 1rem;
+
+      .progress {
+        display: flex;
+        align-items: center;
+        gap: 1rem;
+
+        .progress-circle {
+          width: 5.625rem;
+          height: 5.625rem;
+          padding: 0.63rem 0.625rem 0.75rem 0.75rem;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+
+          .circle {
+            position: absolute;
+            border-radius: 50%;
+            width: 4.25rem;
+            height: 4.25rem;
+            border: 5px solid $blue;
+            border-left-color: $gray;
+            -webkit-transform: rotate(45deg);
+            transform: rotate(45deg);
+          }
+        }
+
+        .percentage {
+          h4 {
+            font-size: 2rem;
+            font-weight: 400;
+            color: $black;
+          }
+
+          p {
+            opacity: 0.5;
+            font-size: 0.875rem;
+            color: $black;
+          }
+        }
+      }
+
+      .progress-line {
+        width: 0.125rem;
+        height: 4.375rem;
+        margin: 0 3.688rem 0 4.938rem;
+        background-color: $gray;
+      }
+    }
+  }
+}
+
+.horizontal-line {
+  height: 0.125rem;
+  background-color: #e8e8e8;
+  width: 85%;
+  margin: 2.5rem 0 1.813rem;
+}
+
+// // Form styles
+// .form {
+//   display: flex;
+//   flex-direction: column;
+//   align-items: flex-start;
+//   width: 85%;
+//   gap: 1rem;
+//   margin-bottom: 20rem;
+
+//   h2 {
+//     font-size: 1.25rem;
+//     font-weight: 800;
+//     text-transform: uppercase;
+//     color: $black;
+//     opacity: 0.5;
+//   }
+
+//   .container {
+//     display: flex;
+//     gap: 2rem;
+//     width: 100%;
```

Comment:
- [ ] Kindly delete all these commented lines of code. It is not professional and you can always use git history if you want to revisit some old code.

Reply:
ooh I forgot to remove these, thanks @t-yanick.